imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -436,62 +443,87 @@ num_completed:
   FIELDNAM: Number of spins completed
   LABLAXIS: number of spins completed
 
-acq_end:
+acq_start_sec:
+  <<: *default_uint32
+  DEPEND_0: epoch
+  VAR_TYPE: support_data
+  CATDESC: Acquisition Start
+  FIELDNAM: Acquisition Start Seconds
+  UNITS: ns
+  LABLAXIS: acquisition start seconds
+
+acq_start_subsec:
+  <<: *default_uint32
+  DEPEND_0: epoch
+  VAR_TYPE: support_data
+  CATDESC: Acquisition Start
+  FIELDNAM: Acquisition Start Sub-Seconds
+  LABLAXIS: acquisition start sub-seconds
```

Comment:
Do we need a UNITS: field to indicate that these are in `ms` (or whatever unit they are).

Reply:
Yeah I'll need to double check on what this should be